dit/divergences/tests/test_generalized_divergences.py

```diff
@@ -96,11 +96,11 @@ def test_divergences_to_kl():
         for dist1 in dists:
             for dist2 in dists:
                 assert_almost_equal(alpha_divergence(dist1, dist2, alpha=-1), kullback_leibler_divergence(dist2, dist1))
-                if dist1 != dist2:
+                if dist1 is not dist2:
```

Comment:
no, we want something like itertools.product(dists, repeat=2) that skips the collision. pairwise would only return "adjacent" pairs.